README.md

```diff
@@ -53,53 +53,54 @@ The installation process for ElixirLS depends on your editor.
 <details>
   <summary>VSCode</summary>
 
-  Please install the extension via the following link: https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls
+Please install the extension via the following link: https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls
+
 </details>
 
 <details>
   <summary>Emacs Installation Instructions</summary>
 
-  Download the latest release:
-  https://github.com/elixir-lsp/elixir-ls/releases/latest and unzip it into a
-  directory (this is the directory referred to as the
-  `"path-to-elixir-ls/release"` below)
-
-  If using `lsp-mode` add this configuration:
-
-  ```elisp
-    (use-package lsp-mode
-      :commands lsp
-      :ensure t
-      :diminish lsp-mode
-      :hook
-      (elixir-mode . lsp)
-      :init
-      (add-to-list 'exec-path "path-to-elixir-ls/release"))
-  ```
+Download the latest release:
+https://github.com/elixir-lsp/elixir-ls/releases/latest and unzip it into a
+directory (this is the directory referred to as the
+`"path-to-elixir-ls/release"` below)
+
+If using `lsp-mode` add this configuration:
+
+```elisp
+  (use-package lsp-mode
+    :commands lsp
+    :ensure t
+    :diminish lsp-mode
+    :hook
+    (elixir-mode . lsp)
+    :init
+    (add-to-list 'exec-path "path-to-elixir-ls/release"))
+```
 
-  For eglot use:
+For eglot use:
 
-  ```elisp
-  (require 'eglot)
+```elisp
+(require 'eglot)
 
-  ;; This is optional. It automatically runs `M-x eglot` for you whenever you are in `elixir-mode`
-  (add-hook 'elixir-mode-hook 'eglot-ensure)
+;; This is optional. It automatically runs `M-x eglot` for you whenever you are in `elixir-mode`
+(add-hook 'elixir-mode-hook 'eglot-ensure)
 
-  ;; Make sure to edit the path appropriately, use the .bat script instead for Windows
-  (add-to-list 'eglot-server-programs '(elixir-mode "path-to-elixir-ls/release/language_server.sh"))
-  ```
+;; Make sure to edit the path appropriately, use the .bat script instead for Windows
+(add-to-list 'eglot-server-programs '(elixir-mode "path-to-elixir-ls/release/language_server.sh"))
+```
 
 </details>
 
 ## Supported versions
 
-Elixir:
-
-- 1.12 minimum
-
-Erlang:
-
-- OTP 22 minimum
+| OTP Versions | Elixir Versions | Supports ElixirLS |                          Issue(s)                          |
+| :----------: | :-------------: | :---------------: | :--------------------------------------------------------: |
+|      22      |   1.12 - 1.13   |        Yes        |         Erlang docs not working (requires EIP 48)          |
+|      23      |   1.12 - 1.14   |        Yes        |                            None                            |
+|      24      |   1.12 - 1.15   |        Yes        |                            None                            |
```

Comment:
```suggestion
|      24      |   1.13 - 1.14   |        Yes        |                            None                            |
```

Reply:
OTP 26 is only supported by Elixir 1.14.5 - 1.15.0 at this time.  In your updates, you're specifying `any` Elixir version for OTP 26.0.  BTW, I referenced the following Elixir docs:

- https://hexdocs.pm/elixir/1.14.5/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp

- https://hexdocs.pm/elixir/1.15.0-rc.1/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp

Reply:
I wanted to express the fact that 1.15 is not supported yet. Do you think it can be improved?

Reply:
I understand.  Do you think that we can simply use `none` instead of `any` in regard to the Elixir version?